### ISSUE TITLE
tabindex is required for the div container of "Command Palette"

### ISSUE THREAD

**Issue by jooyoungseo** [rstudio/rstudio#6962]
### System details

    RStudio Edition : Server
    RStudio Version : 1.4.390
    OS Version      : any
    R Version       : any

### Steps to reproduce the problem

1. With JAWS or other screen reader loaded, press `Ctrl + Shift + P` to open "Command Palette."
2. See if screen readers activate their virtual cursor (AKA browse mode).

### Describe the problem in detail

Virtual cursor/browse mode is activated by default in the command palette, which is not ideal.

This is due in part to the missing `tabindex` attribute in the far parent div container of the command palette; I have confirmed that VSCode command palette has this attribute assigned and has no issue like this.

Please add `tabindex = "-1"` to the div denoted with `class = "gwt-PopupPanel rstudio-themes-scrollbars rstudio-themes-default"`.

### Describe the behavior you expected

Screen readers' browse mode should not be activated when landed in the command palette.

- [x] I have read the guide to submitting good bug reports at https://github.com/rstudio/rstudio/wiki/Writing-Good-Bug-Reports . 
- [x] I have installed the latest version of RStudio and confirmed that the issue still persists.
- [x] If I am reporting a RStudio crash, I have included a diagnostics report. https://support.rstudio.com/hc/en-us/articles/200321257-Running-a-Diagnostics-Report
- [x] I have done my best to include a minimal, self-contained set of instructions for consistently reproducing the issue.

**Comment by gtritchie** [comment]
I'm still seeing inconsistent behavior even with that change, assigning to myself for more investigation.

**Comment by gtritchie** [comment]
Tried again with latest Chrome and JAWS, and it is working as expected. The container of the popup does have `tabindex=-1`, and when I activate the Command Palette, JAWS goes into PC cursor mode; verified via <kbd>Alt+Del</kbd> which announced "Forms Mode, PC Cursor active".

There is still a significant performance issue, especially with JAWS, tracked here: #7869

NVDA is also working as expected for me.
